lib/pdk/util/bundler.rb

```diff
@@ -100,6 +115,10 @@ def binstubs!(gems)
           result[:exit_code].zero?
         end
 
+        def gemfile
```

Comment:
So that it can be called in the `ensure_bundle!` class method (outside the public scope of the BundleHelper instance)